archinstall/lib/models/users.py

```diff
@@ -151,6 +149,10 @@ def __eq__(self, other: object) -> bool:
 
 		return self.enc_password == other.enc_password
 
+	def _encrypt(self, plaintext: str) -> str:
+		from ..crypt import crypt_yescrypt
+		return crypt_yescrypt(plaintext)
+
```

Comment:
This looks like you are trying to fix the import cycle you have created with use of the `Password` class.